tt_metal/hw/ckernels/grayskull/metal/llk_api/llk_sfpu/ckernel_sfpu_zeros.h

```diff
@@ -0,0 +1,29 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ckernel.h"
+#include "ckernel_defs.h"
+#include "noc_nonblocking_api.h"
+
+using namespace sfpi;
+
+namespace ckernel {
+namespace sfpu {
+
+template <bool APPROXIMATION_MODE, int ITERATIONS = 4>
+inline void calculate_zeros()
+{
+    vFloat fill_val = 0.0f;
```

Comment:
any reason for assigning 0 to vFloat and then using instead of directly assigning to dst_reg?

Reply:
`If everyone is ok with this as an acceptable solution, we will implement this as a generalised fill unary op with a fill_value parameter`